modules/mergeImageWithTelemetry/mergeImageWithTelemetry.py

```diff
@@ -0,0 +1,84 @@
+from modules.mergeImageWithTelemetry.mergedData import MergedData
+from modules.timestamp.timestamp import Timestamp
+
+import datetime
+import numpy.typing as npt
+import typing
+
+class MergeImageWithTelemetry:
+    """
+    Merges image pipeline with telemetry pipeline
+    ...
+
+    Attributes
+    ----------
+    curData : Timestamp 
+        the current telemetry data 
+    start : bool
```

Comment:
Use a descriptive modifier (e.g. hasStarted, hasNotStarted, isStarted, etc.). In general all boolean variables should have them.